inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp

```diff
@@ -108,6 +108,7 @@ const Transformations& getDefaultTransformations() {
         {ngraph::opset3::Convert::type_info,           dynamicToStaticUnaryElementwise},
         {ngraph::opset3::Clamp::type_info,             dynamicToStaticUnaryElementwise},
         {ngraph::opset3::Floor::type_info,             dynamicToStaticUnaryElementwise},
+        {ngraph::opset3::Ceiling::type_info,           dynamicToStaticUnaryElementwise},
```

Comment:
It's better to use the latest opset `opset5`.

Reply:
Fixed